### PR TITLE
Fixing issue with Date Range

### PR DESCRIPTION
**Pull request by abgoswam**
This PR addresses issues related to mismatch of data range.  

With this fix, we now see missing events 0 for an example  run with TR dataset where previously we were seeing ~1K missing events

### CODE REVIEW

File: DataScience/DataScience.py [hunk 1]
```diff
@@ -34,6 +34,10 @@
         print("Start and end dates are expected. Example: python datascience.py 20161122 20161130")
     start_date_string = sys.argv[1]
     start_date = date(int(start_date_string[0:4]), int(start_date_string[4:6]), int(start_date_string[6:8]))
+    
+    # We will populate some data structures with info from previous day
+    start_date_prev = start_date + timedelta(days = -1)
```
Comment (eisber) [inline DataScience/DataScience.py]: can you move this as a parameter to the config file? This is actually deployment dependent...

File: DataScience/common.py [hunk 2]
```diff
@@ -154,13 +154,13 @@ def json(self):
...
         return JoinedDataReader(self)
 
-class Trackback(CachedBlob):
-    def __init__(self, block_blob_service, ts, root, blob):
-        super(Trackback,self).__init__(block_blob_service, root, 'onlinetrainer', blob)
-        self.ts = ts
-        self.blob = blob
-        self.ids = []
-        self.data = []
+#class Trackback(CachedBlob):
```
Comment (eisber) [inline DataScience/common.py]: why?

**Comment by eisber** [comment]
thanks! can you address the comments?

File: DataScience/common.py (hunk 2 above)
Comment (abgoswam) [inline DataScience/common.py]: I didn't see it being used anywhere... Is this stub code for future use ?

File: DataScience/DataScience.py (hunk 1 above)
Comment (abgoswam) [inline DataScience/DataScience.py]: not sure what you mean by deployment dependent ?  what kind of parameter do u have in mind .. something like 'usePrevDay=True (default)' ?

**Comment by eisber** [comment]
The additional amount of events we have to load depends on the experimental unit duration. For Track Revenue it’s 20h (or so).

Maybe experimental_unit_duration = 1
An interpret the number as days?

**Comment by eisber** [comment]
Oh, then just remove.

File: DataScience/DataScience.py (hunk 1 above)
Comment (eisber) [inline DataScience/DataScience.py]: The additional amount of events we have to load depends on the experimental unit duration. For TR it’s 20h (or so). 

Maybe experimental_unit_duration = 1 
An interpret the number as days?

**Pull request merged**